tests/integration/docusaurus/connecting_to_your_data/cloud/azure/spark/configured_python_example.py

```diff
@@ -0,0 +1,94 @@
+import os
+from typing import List
+
+from ruamel import yaml
+
+import great_expectations as ge
+from great_expectations.core.batch import Batch, BatchRequest
+
+credential = os.getenv("AZURE_ACCESS_KEY", "")
+
+context = ge.get_context()
+
+datasource_config = {
+    "name": "my_azure_datasource",
+    "class_name": "Datasource",
+    "execution_engine": {
+        "class_name": "PandasExecutionEngine",
+        "azure_options": {
+            "credential": "<YOUR_CREDENTIAL>",
+        },
+    },
+    "data_connectors": {
+        "configured_data_connector_name": {
+            "class_name": "ConfiguredAssetAzureDataConnector",
+            "azure_options": {
+                "credential": "<YOUR_CREDENTIAL>",
+            },
+            "container": "superconductive-public",
+            "name_starts_with": "data/taxi_yellow_trip_data_samples/",
```

Comment:
Could we obfuscate these with `<YOUR_X_HERE>`? I don't know if I did it for the other example but it would make it more applicable to users when we eventually write the docs.

Could also push that off to when we write docs I'll leave it up to you.

Reply:
@cdkini I followed the pattern already in place.  If you agree that this is so, how about we address at a later date?  Thanks!

Reply:
@cdkini This is Python -- so we adhere to the explicit pattern, since an easy string substitution is not available.

Reply:
Should be a dictionary update no? I'm quite confident our approach is substitution in all other connectors so I'd like to push on this. Let me know if you want to discuss.

Reply:
Done.